app/models/services/crossref.rb

```diff
@@ -0,0 +1,17 @@
+class Crossref
+  def initialize(options={})
+    @base_url = 'http://api.crossref.org/'
+    @metadata_url = @base_url + "works/"
+    @funder_url = @base_url + "funders/"
+  end
+
+  def get_doi_metadata(doi)
+    metadata_uri = URI.parse(@metadata_url+doi)
+    begin
+      JSON.parse Net::HTTP.get(metadata_uri)
+    rescue JSON::ParserError => e
+      return false
```

Comment:
Another convention to keep in mind is that it makes more sense if certain methods have certain return values. For instance, a method that returns `false` should also return `true`. In the case that it doesn't return a boolean, then perhaps its negative value should be `nil`.